lib/cocina/models/builders/title_builder.rb

```diff
@@ -245,7 +255,7 @@ def rebuild_structured_value(cocina_title)
                        end
             when 'subtitle'
               result = if !add_punctuation?
-                         [result, value].select(&:presence).join(' ')
+                         [result, value.sub(/^:/, '').strip].select(&:presence).join(' ')
```

Comment:
addresses one spec I copied over;  my new title variants required this tweak;  the old method for getting a display title was unaffected.